client/src/components/pages/PageNotFound/style.css

```diff
@@ -0,0 +1,41 @@
+.not-found {
+    min-height: calc(100vh - calc(129px + 70px));
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.not-found--msg {
+    display: block;
+    font-size: 26px;
+    font-family: 'Merriweather', serif;
+    color: #333;
```

Comment:
give it padding-top. It is too close from the header.
![Screenshot from 2019-05-30 12-30-54](https://user-images.githubusercontent.com/35868173/58623597-0382e400-82d7-11e9-97f8-83fa510b3f4f.png)